lib/settings_base.py

```diff
@@ -525,6 +526,7 @@ def JINJA_CONFIG():
             'css/impala/compat.less',
             'css/impala/localizers.less',
             'css/impala/fxa-migration.less',
+            'css/restyle.css',
```

Comment:
Was there a reason to no use less for this. There might be some opportunity to dry things up a little with vars etc.